shared/dataflow/codeql/dataflow/VariableCapture.qll

```diff
@@ -585,11 +585,13 @@ module Flow<LocationSig Location, InputSig<Location> Input> implements OutputSig
     2 <= strictcount(CapturedVariable v | captureAccess(v, c))
     or
     // Constructors that capture a variable may assign it to a field, which also
-    // entails a this-to-this summary.
-    captureAccess(_, c) and c.isConstructor()
+    // entails a this-to-this summary. If there are multiple constructors, then
```

Comment:
Do we have a test case for the one-constructor-calls-another case you describe?

Reply:
I'll add one.